packages/manager/apps/hpc-vmware-managed-vcd/public/translations/hpc-vmware-managed-vcd/datacentres/storage/Messages_fr_FR.json

```diff
@@ -0,0 +1,11 @@
+{
+  "managed_vcd_vdc_storage_CTA": "Commander de la capacité de stockage",
+  "managed_vcd_vdc_storage_id": "ID",
+  "managed_vcd_vdc_storage_name": "Nom",
+  "managed_vcd_vdc_storage_profile": "Profil",
+  "managed_vcd_vdc_storage_type": "Type",
+  "managed_vcd_vdc_storage_capacity": "Taille",
+  "managed_vcd_vdc_storage_billing": "Paiement",
```

Comment:
Should not be `Facturation` rather ?

Reply:
According to Jira it's not, but it may be better indeed 👍